### ISSUE TITLE
http request issues when calling any endpoint in chrome

### ISSUE THREAD

**Issue by sam-g-roberts** [jadjoubran/laravel5-angular-material-starter#334]
I'm not sure if this is a bug with the library but whislt working on my site today i started getting the below error when trying to make any api call.

`angular.js:13642 Error: Failed to execute 'setRequestHeader' on 'XMLHttpRequest': 'Bearer <script> Sfdump = window.Sfdump || (function (doc) { var refStyle = doc.createElement('style'), rxEsc = /([.*+?^${}`

I'm completely stumped as to whats happened. Currently i've just added the basic angular $http example to rule out that it had anything to do with restangular.

This is my current code for the login component js where the issue is occuring.

```
    login() {
        let user = {
            email: this.email,
            password: this.password
        };
        console.log(user);

        this.$http({
            method: 'GET',
            url: '/someUrl'
        }).then(function successCallback(response) {
            console.log('sucecss');
        }, function errorCallback(response) {
            console.log(response);
        });
    }
```

It's probabbly worth noting that everything works fine in firefox and this is only happening for me in chrome. I've closed down my browser and re-opened it to no avail.

Any help would be deeply appreciated even if it's not library related.

**Comment by sam-g-roberts** [comment]
It's probabbly worth noting that everything works fine in firefox and this is only happening for me in chrome. I've closed down my browser and re-opened it to no avail.

**Comment by sam-g-roberts** [comment]
The plot thickens. It works in chrome if i add https before my local url

**Comment by jadjoubran** [comment]
Hi @sam-g-roberts 
I think this has been solved before here #298 

Let me know if it works for you 😄
